include/wil/common.h

```diff
@@ -305,6 +305,29 @@ static_assert(WIL_EXCEPTION_MODE <= 2, "Invalid exception mode");
 #error Must enable exceptions when WIL_EXCEPTION_MODE == 1
 #endif
 
+#ifdef WIL_DOXYGEN

```

Comment:
Why does the DOXYGEN define have anything to do with STL usage?  On closer inspection this seems to be "if in doxygen mode use STL".  Maybe a comment would help?

Reply:
The Doxygen define just means "Doxygen is running" - it's more of a short-circuit so that Doxygen provides the targeted code in the generated documentation. In this case, the desire is for the following comment to be present in the documentation. The point here of providing a separate ifdef for Doxygen is so that we don't have to worry about _which_ "path" Doxygen evaluates & put the comment there